docs/src/components/showcase/ThemeSlider.tsx

```diff
@@ -70,7 +70,6 @@ export default function ThemeSlider() {
                 fontWeight: 'semiBold',
               },
               [`& .${sliderClasses.thumb}`]: {
-                mx: -1,
```

Comment:
@danilo-leal everything seems to work just fine, but let me know if there was another reason to include this.